cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -290,6 +295,7 @@ func (s *Server) Run() {
 
 			// Stop indexing repos we don't need to track anymore
 			count := s.queue.MaybeRemoveMissing(repos.IDs)
+			metricNumStoppedTracking.Set(float64(count))
```

Comment:
I think this would be more approriate as a counter. This will depend on making sure prometheus catches the gauge value. However with a counter we can use `delta(metric_name[10m])` to find the number of repos we stopped tracking in the last ten minutes.

Reply:
alright, switched to counter.